packages/plugins/integration-zapier-ui/src/lib/components/zapier-settings/zapier-settings.component.html

```
@@ -0,0 +1,48 @@
+					'INTEGRATIONS.ZAPIER_PAGE.SETTINGS.WEBHOOK_URL' | translate
+				}}</label>
+				<input
+					fullWidth
+					id="webhookUrl"
+					formControlName="webhookUrl"
+					type="url"
+					nbInput
+					[placeholder]="'INTEGRATIONS.ZAPIER_PAGE.SETTINGS.WEBHOOK_URL_PLACEHOLDER' | translate"
+				/>
+				<div
+					*ngIf="form.get('webhookUrl').touched && form.get('webhookUrl').errors?.required"
```

HTMLHint (1.5.0)

[error] The attribute name of [ *ngIf ] must be in lowercase.

(attr-lowercase)